SoS/SoS.tp2

```diff
@@ -1117,6 +1317,7 @@ BEGIN @21011
             ~%MOD_FOLDER%/Wed/Alt/AR4230N.WED~ ~override~
 
     COPY    ~%MOD_FOLDER%/Are/Alt/AR4230.are~ ~override~
+		PATCH_INCLUDE ~%MOD_FOLDER%/lib/install-nightmaps.tpa~
```

Comment:
Hello @weigo87 Are you sure that this is also required for these alternative areas created by Cuttooth? He added his own night version of the area so it might be just correct.

Reply:
This little lib does three things and is from the BGGO:
1. checks whether it is an outdoor and then sets the extended. (we don't really need it here)
2. switches off the animations of animals overnight.
3. installs new animations for torches and candles

Of course you could have installed it manually, but I didn't change anything in the background graphics.

Reply:
OK, thanks